venter/admin.py

```diff
@@ -32,14 +36,98 @@ class CommentModelAdmin(admin.ModelAdmin):
     list_display = ['__str__', 'complaint', 'time']
     model = Comment
 
+class AuthoritiesModelAdmin(admin.ModelAdmin):
+    list_display = ['name', '__str__']
+    model = Authorities
+
 class ComplaintModelAdmin(admin.ModelAdmin):
     readonly_fields = ['created_by']
-    list_display = ['report_date', 'status']
-    list_editable = ['status']
+    list_display = ['report_date', 'status', 'authority_email']
+    list_editable = ['status', 'authority_email']
     list_filter = ['status']
     inlines = [CommentTabularInline, TagTabularInline, UserLikedTabularInline, ComplaintMediaTabularInline]
     exclude = ('tags', 'users_up_voted', 'media',)
     search_fields = ['status', 'description', 'created_by__name']
+    actions = ['mark_as_resolved', 'mark_as_in_progress', 'mark_as_deleted', 'send_emails']
+
+    def mark_as_resolved(self, request, queryset):  # pylint: disable=R0201
```

Comment:
You can fix R0201 by adding a `@staticmethod` decorator and removing the `self` argument. See [this wiki](http://pylint-messages.wikidot.com/messages:r0201)